### PR TITLE
New login screen to make non-master ticket logins easier to navigate

### PR DESCRIPTION
**Pull request by jyng**
Screen Shot 2020-03-09 at 7 40 59 PM

image

Screen Shot 2020-03-09 at 7 40 17 PM

### CODE REVIEW

**Comment by matildepark** [comment]
Surely just trailing ellipses from “Hardware Wallet…” is better than “etc.” in a button?

— 
~haddef-sigwen
https://urbit.org

**Comment by galenwp** [comment]
Yeah 'etc...' seems a bit odd. 'Other options (Metamask, Ledger, etc)' or similar.

**Comment by vvisigoth** [comment]
YEAH
—
~poldec-tonteg
http://urbit.org

**Comment by vvisigoth** [comment]
Otherwise, i think this is vast improvement, thnx. Approved based on changing ellipsis.

**Comment by jyng** [comment]
Fixed

**Comment by jyng** [comment]
Edited

File: src/views/Login.js [hunk 1]
```diff
@@ -61,37 +59,54 @@ export default function Login() {
...
+            </Grid.Item>
+          </>
+        )}
+        {isOther && (
+          <Grid.Item
+            as={LinkButton}
+            onClick={() => setisOther(false)}
+            full
+            className="t-center underline f6 mt8">
+            Back
```
Comment (Fang-) [inline src/views/Login.js]: Should this just say "Master Ticket" or similar instead, to have more symmetry with the "Metamask, Mnemonic..." button?

File: src/views/Login.js [hunk 2]
```diff
@@ -61,37 +59,54 @@ export default function Login() {
...
+              as={Button}
+              className="b-solid b1 b-black"
+              center
+              onClick={() => setisOther(true)}>
+              Metamask, Mnemonic, Hardware Wallet...
+            </Grid.Item>
+            <Grid.Item full onClick={goToActivate} className="mv10 t-center f6">
+              <span className="gray4">New Urbit ID? </span>
+              <LinkButton>Activate</LinkButton>
+            </Grid.Item>
```
Comment (Fang-) [inline src/views/Login.js]: Not sure why we don't just show this on both pages (ie regardless of `isOther`), but I can see how it's only strictly needed for the landing conditions.

(Also the space after "ID?" bother me. (^: )

File: src/views/Login/Ticket.js [hunk 3]
```diff
@@ -168,71 +227,58 @@ export default function Ticket({ className, goHome }) {
...
 
             <Grid.Item full as={FormError} />
 
-            <Grid.Item full as={SubmitButton} handleSubmit={handleSubmit}>
+            <Grid.Item
+              full
+              as={SubmitButton}
+              center
+              handleSubmit={handleSubmit}>
               {isWarning =>
```
Comment (Fang-) [inline src/views/Login/Ticket.js]: Maybe this is as good a time as any to tear this out. See also #405. (But we might want further discussion on that.)

Comment (jyng) [reply src/views/Login/Ticket.js]: Yeah while I was styling this, I felt like it was an unnecessary layer of abstraction, but what do I know lol

File: src/views/Login.js (hunk 2 above)
Comment (jyng) [reply src/views/Login.js]: Yeah, this crossed my mind, but I felt like this page is for advanced users who are familiar with crypto. On the flipside, we haven't done a good job of getting people to use Master Ticket (even if they bought a planet from a third-party)

**Pull request merged**